httpasyncclient/src/main/java/com/huwei/devcloud/httpasyncclient/BackController.java

```diff
@@ -0,0 +1,41 @@
+package com.huwei.devcloud.httpasyncclient;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import org.apache.log4j.Logger;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.data.mongo.MongoDataAutoConfiguration;
+import org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@EnableAutoConfiguration(exclude = {MongoAutoConfiguration.class, MongoDataAutoConfiguration.class})

```

Comment:
Does the project  need the  `MongoAutoConfiguration.class` and `MongoDataAutoConfiguration.class` configuration class? if not, please remove it